sites/kit.svelte.dev/scripts/types/index.js

```diff
@@ -5,7 +5,14 @@ import prettier from 'prettier';
 import { mkdirp } from '../../../../packages/kit/src/utils/filesystem.js';
 import { fileURLToPath } from 'url';
 
-/** @typedef {{ name: string; comment: string; markdown: string; }} Extracted */
+/** @typedef {{
+ * name: string;
+ * comment: string;
+ * markdown?: string;
+ * snippet: string;
+ * deprecated: string | null;
+ * children: Extracted[] }
+ * } Extracted */
```

Comment:
I expect VS Code won't have the proper syntax highlighting unless you tweak it:
```suggestion
/**
 * @typedef {{
 * name: string;
 * comment: string;
 * markdown?: string;
 * snippet: string;
 * deprecated: string | null;
 * children: Extracted[] }
 * } Extracted
 */
```